logstash-core/lib/logstash/agent.rb

```diff
@@ -176,18 +176,18 @@ def create_pipeline(settings, config=nil)
       begin
         config = fetch_config(settings)
       rescue => e
-        @logger.error("failed to fetch pipeline configuration", :message => e.message)
+        @logger.error("failed to fetch pipeline configuration", "message" => e.message)
         return
       end
     end
 
     begin
       LogStash::Pipeline.new(config, settings, metric)
     rescue => e
-      if @logger.debug?
-        @logger.error("fetched an invalid config", :config => config, :reason => e.message, :backtrace => e.backtrace)
+      if @logger.is_debug_enabled
```

Comment:
this was here with the original intent of not having a light wrapper around Log4j's logger object. In the upcoming refactor, these changes will be reverted as I will introduce a `#debug?` method to the future wrapper